numpyro/distributions/util.py

```diff
@@ -346,6 +346,17 @@ def clamp_probs(probs):
     return jnp.clip(probs, a_min=finfo.tiny, a_max=1. - finfo.eps)
 
 
+def is_identically_one(x):
+    """
+    Check if argument is exactly the number one. True for the number one;
+    false for other numbers; false for ndarrays.
+    """
+    if isinstance(x, (int, float)):
```

Comment:
Do we need to check for numpy scalars separately?

Reply:
I am not so sure how to check that properly because some numpy scalars are already subclasses of `int, float`, some are not. This simplifies stuff under `plate`, where `scale = 1.` for most of the situations.